rule/unused-import-alias.go

```diff
@@ -0,0 +1,52 @@
+package rule
+
+import (
+	"fmt"
+	"go/ast"
+	"strings"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// UnusedImportAlias lints given else constructs.
+type UnusedImportAlias struct{}
+
+// Apply applies the rule to given file.
+func (*UnusedImportAlias) Apply(file *lint.File, _ lint.Arguments) []lint.Failure {
+	var failures []lint.Failure
+
+	for _, imp := range file.AST.Imports {
+		if imp.Name == nil {
+			continue
+		}
+
+		if getImportPackageName(imp) == imp.Name.Name {
+			failures = append(failures, lint.Failure{
+				Confidence: 1,
+				Failure:    fmt.Sprintf("Import alias \"%s\" is not used", imp.Name.Name),
+				Node:       imp,
+				Category:   "imports",
+			})
+		}
+	}
+
+	return failures
+}
+
+// Name returns the rule name.
+func (*UnusedImportAlias) Name() string {
+	return "unused-import-alias"
```

Comment:
What about `redundant-import-alias` ?

Reply:
Yes, i think its more accurate to call it "redundant"